NineAnimator/Models/Anime Source/animeunity.it/AnimeUnity+Anime.swift

```diff
@@ -21,91 +21,85 @@ import Foundation
+                    ), "")
                 }
-                
                 // Information
-                var animeSynopsis = ""
+                let synopsis = try bowl.select("div.description").text()
+                // Attributes
                 var additionalAttributes = [Anime.AttributeKey: Any]()
-                _ = try bowl.select("div.card-body p").compactMap {entry -> String in
-                    let trama = ""
-                    if try entry.text().contains("TRAMA") {
-                        let trama = try entry.text()
-                        animeSynopsis = String(trama.dropFirst(7))
-                    }
-                    if try entry.text().contains("TITOLO") {
-                        let title = try entry.text()
-                        animeTitle = String(title.dropFirst(8))
+                _ = try bowl.select("div.info-item").compactMap {entry -> Void in
+                    if try entry.text().contains("Anno") {
+                        let year = try entry.select("small").text()
+                        additionalAttributes[.airDate] = year
                     }
-                    if try entry.text().contains("ANNO DI USCITA:") {
-                        let airdate = try entry.text()
-                        additionalAttributes[.airDate] = String(airdate.dropFirst(16))
-                    }
-                    return trama
-                }
-                // Attributes
-                
-                let detailContainers = try bowl.select("div.info div.detail")
-                
-                for container in detailContainers {
-                    let attributeName = ""
-                    let attributeValue = try container
-                        .text()
-                        .trimmingCharacters(in: .whitespacesAndNewlines)
-                    if attributeName.lowercased().contains("release date") {
-                        additionalAttributes[.airDate] = attributeValue
+                    if try entry.text().contains("Valutazione") {
+                        let val = try entry.select("small").text()
+                        let rating = (val as NSString).floatValue
+                        additionalAttributes[.rating] = rating
+                        additionalAttributes[.ratingScale] = Float(10.0)
                     }
                 }
                 return Anime(
                     reconstructedAnimeLink,
-                    
-                    alias: animeTitle,
+                    alias: "",
                     additionalAttributes: additionalAttributes,
-                    description: animeSynopsis,
+                    description: synopsis,
                     on: [ NASourceAnimeUnity.AnimeUnityStream: "AnimeUnity" ],
-                    episodes: [ NASourceAnimeUnity.AnimeUnityStream: episodes ],
-                    episodesAttributes: [:]
+                    episodes: [ NASourceAnimeUnity.AnimeUnityStream: episodesList.map { $0?.0 as! EpisodeLink} ]
```

Comment:
Please avoid using force casting. Use `compactMap` or similar instead if the element is optional.